src/hooks/useDclFeatureFlags.ts

```diff
@@ -0,0 +1,27 @@
+import { useCallback } from 'react'
+
+import { FeatureFlagOptions, fetchFlags } from '@dcl/feature-flags'
+import { useQuery } from '@tanstack/react-query'
+
+export default function useFeatureFlags(options = { applicationName: ['dao', 'dapps'] }) {
```

Comment:
`dao` is Governance name for foundation right?

Reply:
I guess so...